Tests/SwiftDocCTests/Infrastructure/ExternalReferenceResolverTests.swift

```diff
@@ -616,32 +601,30 @@ class ExternalReferenceResolverTests: XCTestCase {
     
     // Tests that external references are included in task groups, rdar://72119391
     func testResolveExternalReferenceInTaskGroups() throws {
-        // Copy the test bundle and add external links to the MyKit Topics.
-        let workspace = DocumentationWorkspace()
-        let (tempURL, _, _) = try testBundleAndContext(copying: "TestBundle")
-        
-        try """
-        # ``MyKit``
-        MyKit module root symbol
-        ## Topics
-        ### Task Group
-         - <doc:article>
-         - <doc:article2>
-         - <doc://com.external.testbundle/article>
-         - <doc://com.external.testbundle/article2>
-        """.write(to: tempURL.appendingPathComponent("documentation").appendingPathComponent("mykit.md"), atomically: true, encoding: .utf8)
-        
-        // Load the new test bundle
-        let dataProvider = try LocalFileSystemDataProvider(rootURL: tempURL)
-        guard let bundle = try dataProvider.bundles().first else {
-            XCTFail("Failed to create a temporary test bundle")
-            return
-        }
-        try workspace.registerProvider(dataProvider)
-        let context = try DocumentationContext(dataProvider: workspace)
+        let resolver = TestMultiResultExternalReferenceResolver()
+        resolver.entitiesToReturn = [
+            "/article": .success(.init(referencePath: "/externally/resolved/path/to/article")),
+            "/article2": .success(.init(referencePath: "/externally/resolved/path/to/article2")),
+            
+            "/externally/resolved/path/to/article": .success(.init(referencePath: "/externally/resolved/path/to/article")),
+            "/externally/resolved/path/to/article2": .success(.init(referencePath: "/externally/resolved/path/to/article2")),
+        ]
         
-        // Add external resolver
-        context.externalDocumentationSources = ["com.external.testbundle" : TestExternalReferenceResolver()]
```

Comment:
This is another example of a test that with the bug of making configuration changes after the context has "registered" the documentation, thus having no effect and verifying that the links _aren't_ resolved.